src/guide/jbs-jdk-bug-system.md

```diff
@@ -72,7 +72,51 @@ A few things to keep in mind when filing an issue:
 * Always file separate bugs for different issues.
   * If two crashes look related, but not similar enough to be sure they are the same, it's easier to later close a bug as a duplicate than it is to separate out one bug into two.
 
-Things to keep in mind when requesting an improvement:
+#### Indicating what releases an issue is applicable to
+
+Knowing when an issue was introduced is important to determine the impact of the issue and where it needs to be resolved. While in some cases it may be clear, it is likely that on submission and during triage this won't be known, instead we will have one or two data points from which we can begin to understand the range of releases which the issue impacts.
+
+The [Affects Version]{.jbs-field} field is used to indicate which releases an issue is applicable to, and to avoid having to set it to an exhaustive list of impacted releases the following assumptions are used to give that range:
+
+1) If a issue applicable to a feature release N it is assumed to be applicable to all (more recent) releases unless indicated otherwise (see (Rel)-na below).
```

Comment:
```suggestion
1) If an issue applicable to a feature release N, it is assumed to be applicable to all (more recent) releases unless indicated otherwise (see (Rel)-na below).
```

Reply:
fixed